app/jobs/regular/post_alert.rb

```diff
@@ -3,7 +3,7 @@ class PostAlert < Jobs::Base
 
     def execute(args)
       # maybe it was removed by the time we are making the post
-      if post = Post.find_by(id: args[:post_id])
+      if post = Post.find(args[:post_id])
```

Comment:
Actually, wouldn't this alone cause it to work? Because Sidekiq automatically retries failed jobs. It'd just inflate the failed jobs count.